dotnet/src/SemanticKernel.Abstractions/Schema/JsonSchemaFunctionParameters.cs

```diff
@@ -0,0 +1,34 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System.Collections.Generic;
+using System.Text.Json;
+using System.Text.Json.Serialization;
+
+#pragma warning disable IDE0130
+// ReSharper disable once CheckNamespace - Using the main namespace
+namespace Microsoft.SemanticKernel;
+#pragma warning restore IDE0130
+
+/// <summary>
+/// A class to describe the parameters of an SKFunction in a Json Schema friendly way.
+/// </summary>
+public sealed class JsonSchemaFunctionParameters
+{
+    /// <summary>
+    /// The type of schema which is always "object" when describing function parameters.
+    /// </summary>
+    [JsonPropertyName("type")]
+    public string Type => "object";
```

Comment:
Why is this needed if it never changes?

If it stays should it be called `ParameterType`?

Reply:
This is fixed value because we're describing the function as an object and its parameters as properties on the object. This is the defined format that OpenAI expects. The json name needs to be type per the OpenAI spec, but I could change the C# property name if you think that would make more sense.


Example:
```
{
        "name": "search_hotels",
        "description": "Retrieves hotels from the search index based on the parameters provided",
        "parameters": {
            "type": "object",
            "properties": {
                "location": {
                    "type": "string",
                    "description": "The location of the hotel (i.e. Seattle, WA)"
                },
                "max_price": {
                    "type": "number",
                    "description": "The maximum price for the hotel"
                },
                "features": {
                    "type": "string",
                    "description": "A comma separated list of features (i.e. beachfront, free wifi, etc.)"
                }
            },
            "required": ["location"]
        }
    }
```